services/nomad/mirror/sync.nomad

```diff
@@ -55,5 +62,42 @@ EOF
         destination = "/mirror"
       }
     }
+
+    task "sources-rsync" {
+      leader = true
```

Comment:
this might not be right

Reply:
In this case you probably don't need to set a leader at all.  A leader is set when a clear relationship exists between one task and a group of dependent workers, which is not the case here.